README.md

```diff
@@ -109,10 +109,12 @@ The content of that file should be:
 This will run every minute as the www-data user and check if there are any
 outstanding tasks that needs to be executed.
 
-**Warning**: Observe that the crontask module doesn't to any scheduling. If the
-run time is more than one minute, it might start another process that interferes
-with the still running process. You can either trigger the `dev/cron` task less
-often or use something like [sera](https://github.com/silverstripe-labs/sera).
+**Warning**: Observe that the crontask module doesn't to any checking. If 
+you define a task to run every 5 mins it will run every 5 minutes whether it 
+completed or not (as a normal cron would). If the run time of an 'every-5-minutes' 
+task started at 17:10 is more than five minutes, it might start another process 
+at 17:15 that interferes with the still running process. You can either make 
+the task run less often or use something like [sera](https://github.com/silverstripe-labs/sera).
```

Comment:
1. That's the wrong URL
2. It's an unsupported/abandoned module
3. It's not in packagist (because of above).

I'd recommend using the [queuedjobs module](https://github.com/silverstripe-australia/silverstripe-queuedjobs)